js/app.js

```diff
@@ -4,218 +4,77 @@
-    for (var i = 0; i < hours.length; i++) {
-      this.custPerHour.push(Math.floor(Math.random() * (this.maxCustomer - this.minCustomer + 1) + this.minCustomer));
-      // console.log(this.custPerHour[i]);
-    }
-  },
-  fillCookiesPerHour: function() {
-    for (var j = 0; j < this.custPerHour.length; j++) {
-      this.cookiesPerHour.push(Math.floor(this.custPerHour[j] * this.avgSale));
-      // console.log(this.cookiesPerHour[j]);
-    }
-  },
-  calcTotalCookies: function() {
-    for (var k = 0; k < this.cookiesPerHour.length; k++) {
-      this.totalCookies += this.cookiesPerHour[k];
-      // console.log(this.totalCookies);
-    }
-  },
-  render: function() {
-    var ulEl = document.getElementById('Sea Center');
-    for(var l = 0; l < hoursLong.length; l++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = hoursLong[l] + ': ' + this.cookiesPerHour[l] + ' cookies';
-      ulEl.appendChild(liEl);
-    } var li2El = document.createElement('li');
-    li2El.textContent = 'Total: ' + this.totalCookies + ' cookies';
-    ulEl.appendChild(li2El);
-  },
-};
-SeaCen.fillCustPerHour();
-SeaCen.fillCookiesPerHour();
-SeaCen.calcTotalCookies();
-SeaCen.render();
+//end of constructor
+//Creation of instances of the constructor
+new StoreBranch('Alki','Alki', 2, 16, 4.6);
+new StoreBranch('Cap Hill', 'Capitol Hill', 20, 38, 2.3);
+new StoreBranch('SeaTac','SeaTac Airport', 3, 24, 1.2);
+new StoreBranch('Sea Center', 'Seattle Center', 11, 38, 3.7);
+new StoreBranch('Pike', '1st and Pike', 23, 65, 6.3);
 
-
-var pike = {
-  name: '1st and Pike',
-  minCustomer: 23,
-  maxCustomer: 65,
-  avgSale: 6.3,
-  custPerHour: [],
-  cookiesPerHour: [],
-  totalCookies: 0,
-  fillCustPerHour: function() {
-    for (var i = 0; i < hours.length; i++) {
-      this.custPerHour.push(Math.floor(Math.random() * (this.maxCustomer - this.minCustomer + 1) + this.minCustomer));
-      // console.log(this.custPerHour[i]);
-    }
-  },
-  fillCookiesPerHour: function() {
-    for (var j = 0; j < this.custPerHour.length; j++) {
-      this.cookiesPerHour.push(Math.floor(this.custPerHour[j] * this.avgSale));
-      // console.log(this.cookiesPerHour[j]);
-    }
-  },
-  calcTotalCookies: function() {
-    for (var k = 0; k < this.cookiesPerHour.length; k++) {
-      this.totalCookies += this.cookiesPerHour[k];
-      // console.log(this.totalCookies);
-    }
-  },
-  render: function() {
-    var ulEl = document.getElementById('Pike');
-    for(var l = 0; l < hoursLong.length; l++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = hoursLong[l] + ': ' + this.cookiesPerHour[l] + ' cookies';
-      ulEl.appendChild(liEl);
-    } var li2El = document.createElement('li');
-    li2El.textContent = 'Total: ' + this.totalCookies + ' cookies';
-    ulEl.appendChild(li2El);
-  },
-};
-pike.fillCustPerHour();
-pike.fillCookiesPerHour();
-pike.calcTotalCookies();
-pike.render();
\ No newline at end of file
+function render2_0() {
```

Comment:
You should name this something different. For one, you should not be using underscores in a variable name. Also, why is this "render version 2"? Doesn't this just render the header row? Maybe you should name this function something to that effect.